src/modal/modal.scss

```diff
@@ -91,6 +91,9 @@
 }
 
 @mixin iui-modal-full-page {
+  --_iui-full-page-modal-slide-in-speed: 0.3s;
```

Comment:
Why you decided to use css vars here? Do you plan to change these values in react? Usually, we use css vars, if we plan to change the values somewhere else (css vars are used in browser scope).
In this case, maybe just simple sass variables are enough?